packages/atoms/src/SearchInput.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useRef } from 'react';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { TextInput, TouchableOpacity, TouchableWithoutFeedback } from 'react-native';
+import { Icon } from './Icon/Icon';
+import { Box } from './Box';
+
+type InputProps = {
+    value: string;
+    onChange: (value: string) => void;
+    placeholder: string;
+    isDisabled?: boolean;
+};
+
+const inputStyle = prepareNativeStyle(utils => ({
+    ...utils.typography.body,
+    flex: 1,
+    color: utils.colors.gray700,
+    marginLeft: utils.spacings.md,
+}));
+
+const clearIconStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.gray500,
+    height: 20,
+    width: 20,
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: utils.borders.radii.round,
+}));
+
+type InputStyleProps = {
+    isFocused: boolean;
+};
+const inputWrapperStyle = prepareNativeStyle<InputStyleProps>((utils, { isFocused }) => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    height: 48,
+    borderWidth: utils.borders.widths.sm,
+    borderColor: utils.colors.gray200,
+    backgroundColor: utils.colors.gray200,
+    borderRadius: utils.borders.radii.basic,
+    paddingLeft: 14,
+    paddingRight: 14.25,
+    extend: [
+        {
+            condition: isFocused,
+            style: {
+                borderColor: utils.colors.gray500,
+            },
+        },
+    ],
+}));
+
+export const SearchInput = ({ value, onChange, placeholder, isDisabled = false }: InputProps) => {
+    const { applyStyle } = useNativeStyles();
+    const [isFocused, setIsFocused] = useState<boolean>(false);
+    const searchInputRef = useRef<TextInput | null>(null);
+
+    const handleClear = () => {
+        // Wanted behaviour for clearing text from input.
```

Comment:
please remove this comment, it's saying thing that is obvious

Reply:
Fixed